pkgs/applications/networking/cluster/habitat/default.nix

```diff
@@ -5,36 +5,68 @@ with rustPlatform;
 
 buildRustPackage rec {
   name = "habitat-${version}";
-  version = "0.30.2";
+  version = "0.62.0";
 
   src = fetchFromGitHub {
     owner = "habitat-sh";
     repo = "habitat";
     rev = version;
-    sha256 = "0pqrm85pd9hqn5fwqjbyyrrfh4k7q9mi9qy9hm8yigk5l8mw44y1";
+    sha256 = "0n7bw0zqk8psylk2xkajahcrb6j51hn2symzlyx0v1zcnzdwzbvj";
   };
 
-  cargoSha256 = "1ahfm5agvabqqqgjsyjb95xxbc7mng1mdyclcakwp1m1qdkxx9p0";
+  cargoSha256 = "17gifxkkg0bbjsbsj72kpzcimqk30pxks2299z758z2ymww5h80g";
 
   buildInputs = [ libsodium libarchive openssl ];
 
   nativeBuildInputs = [ pkgconfig ];
 
   cargoBuildFlags = ["--package hab"];
 
+  postUnpack = ''
+    eval "$cargoDepsHook"
+    unpackFile "$cargoDeps"
+    cargoDepsCopy=$(stripHash $(basename $cargoDeps))
+    chmod -R +w "$cargoDepsCopy"
+    mkdir -p .cargo
+    cat >.cargo/config <<-EOF
```

Comment:
Fortunately we where able to get rid of this in  https://github.com/NixOS/nixpkgs/pull/46362

Can you rebase this package on staging?